contracts/base/Initializable.sol

```diff
@@ -0,0 +1,53 @@
+/*
+    Copyright 2020 Fabrx Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+pragma solidity 0.5.17;
+
+
+// Libraries
+
+// Commons
+
+// Interfaces
+
+contract Initializable {
+    /* State Variables */
+
+    bool public _isInitialized;
```

Comment:
I believe that it is common practice for a variable beginning with `_` tends to be private/internal, and that public variables do not start with _. Do you follow a different convention?

Reply:
You're right. I made it private.